src/pages/api/companies/index.ts

```diff
@@ -7,16 +7,23 @@ import {
 } from '@/lib/messages'
 import type { ApiResponse } from '@/lib/types/api'
 import { createCompany } from '@/useCases/createCompany'
+import { getCompanies } from '@/useCases/getCompanies'
 import type { Company } from '@prisma/client'
 import type { NextApiRequest, NextApiResponse } from 'next'
 
+export type CompanyResponse = {
+  companies?: Company[]
```

Comment:
I think this `companies` should not be optional, because if any errors occurred, then we will have an empty `data` attribute in the response (it's optional by default).